test/index.js

```diff
@@ -137,7 +137,7 @@ describe('onBuild()', () => {
   test('runs next on netlify', async () => {
     await plugin.onBuild({
       constants: {
-        PUBLISH_DIR: '',
+        PUBLISH_DIR: '.',
```

Comment:
This is passed to `make-dir`. Using `.` is closer to production behavior than an empty string (which can never happen in production).